org.idempiere.test/src/org/idempiere/test/model/MProductTest.java

```diff
@@ -231,8 +231,9 @@ public void testRemoveStorageRecords() {
 		product.setIsActive(false);
 		assertThrows(AdempiereException.class, () -> product.saveEx());
 		
-		//clear on hand so that we can deactivate product
+		//clear on hand and reservation so that we can deactivate product
```

Comment:
maybe this change is from another ticket?

Reply:
I just fix the unit test error when I'm testing this ticket. I can move it else where if that's better.

Reply:
no problem - it was just curious